New swiftlint failures

Looks like a new version was shipped out from underneath us and we've got a bunch of new failures happening.
```
Running swiftformat
swiftformat megazords 'components/*/ios' --exclude '**/Generated' --exclude components/nimbus/ios/Nimbus/Utils --lint --swiftversion 5
Running SwiftFormat...
(lint mode - no files will be changed.)
components/places/ios/Places/Places.swift:8:1: warning: (redundantInternal) Remove redundant internal access control.
components/places/ios/Places/Places.swift:9:1: warning: (redundantInternal) Remove redundant internal access control.
components/nimbus/ios/Nimbus/NimbusMessagingHelpers.swift:29:1: warning: (sortTypealiases) Sort protocol composition typealiases alphabetically.
components/nimbus/ios/Nimbus/NimbusMessagingHelpers.swift:64:1: warning: (redundantInternal) Remove redundant internal access control.
components/nimbus/ios/Nimbus/NimbusMessagingHelpers.swift:76:1: warning: (redundantInternal) Remove redundant internal access control.
components/nimbus/ios/Nimbus/Dictionary+.swift:7:1: warning: (redundantInternal) Remove redundant internal access control.
components/nimbus/ios/Nimbus/FeatureVariables.swift:306:1: warning: (redundantInternal) Remove redundant internal access control.
components/nimbus/ios/Nimbus/FeatureVariables.swift:308:1: warning: (redundantInternal) Remove redundant internal access control.
components/nimbus/ios/Nimbus/Nimbus.swift:29:1: warning: (redundantInternal) Remove redundant internal access control.
components/nimbus/ios/Nimbus/Nimbus.swift:30:1: warning: (wrapArguments) Align wrapped function arguments or collection elements.
components/nimbus/ios/Nimbus/Nimbus.swift:31:1: warning: (wrapArguments) Align wrapped function arguments or collection elements.
components/nimbus/ios/Nimbus/Nimbus.swift:154:1: warning: (redundantInternal) Remove redundant internal access control.
components/nimbus/ios/Nimbus/Nimbus.swift:160:1: warning: (redundantInternal) Remove redundant internal access control.
components/nimbus/ios/Nimbus/Nimbus.swift:175:1: warning: (redundantInternal) Remove redundant internal access control.
components/nimbus/ios/Nimbus/Nimbus.swift:185:1: warning: (redundantInternal) Remove redundant internal access control.
components/nimbus/ios/Nimbus/Nimbus.swift:222:1: warning: (redundantInternal) Remove redundant internal access control.
components/nimbus/ios/Nimbus/Nimbus.swift:267:1: warning: (redundantInternal) Remove redundant internal access control.
components/fxa-client/ios/FxAClient/FxAccountState.swift:10:1: warning: (redundantInternal) Remove redundant internal access control.
components/fxa-client/ios/FxAClient/FxAccountState.swift:22:1: warning: (redundantInternal) Remove redundant internal access control.
components/fxa-client/ios/FxAClient/FxAccountStorage.swift:8:1: warning: (redundantInternal) Remove redundant internal access control.
components/fxa-client/ios/FxAClient/FxAccountStorage.swift:9:1: warning: (redundantInternal) Remove redundant internal access control.
components/fxa-client/ios/FxAClient/FxAccountStorage.swift:10:1: warning: (redundantInternal) Remove redundant internal access control.
components/fxa-client/ios/FxAClient/FxAccountLogging.swift:14:1: warning: (redundantInternal) Remove redundant internal access control.
components/fxa-client/ios/FxAClient/FxAccountLogging.swift:18:1: warning: (redundantInternal) Remove redundant internal access control.
components/fxa-client/ios/FxAClient/FxAccountLogging.swift:22:1: warning: (redundantInternal) Remove redundant internal access control.
components/fxa-client/ios/FxAClient/FxAccountDeviceConstellation.swift:133:1: warning: (redundantInternal) Remove redundant internal access control.
components/fxa-client/ios/FxAClient/FxAccountDeviceConstellation.swift:140:1: warning: (redundantInternal) Remove redundant internal access control.
components/fxa-client/ios/FxAClient/FxAccountDeviceConstellation.swift:151:1: warning: (redundantInternal) Remove redundant internal access control.
components/fxa-client/ios/FxAClient/PersistedFirefoxAccount.swift:260:1: warning: (redundantInternal) Remove redundant internal access control.
components/fxa-client/ios/FxAClient/PersistedFirefoxAccount.swift:272:1: warning: (redundantInternal) Remove redundant internal access control.
components/fxa-client/ios/FxAClient/MZKeychain/KeychainItemAccessibility.swift:2:1: warning: (headerFileName) Ensure file name reference in header matches actual file name.
components/fxa-client/ios/FxAClient/FxAccountManager.swift:320:1: warning: (redundantInternal) Remove redundant internal access control.
components/fxa-client/ios/FxAClient/FxAccountManager.swift:343:1: warning: (redundantInternal) Remove redundant internal access control.
components/fxa-client/ios/FxAClient/FxAccountManager.swift:542:1: warning: (redundantInternal) Remove redundant internal access control.
components/fxa-client/ios/FxAClient/FxAccountManager.swift:546:1: warning: (redundantInternal) Remove redundant internal access control.
components/fxa-client/ios/FxAClient/FxAccountManager.swift:550:1: warning: (redundantInternal) Remove redundant internal access control.
components/fxa-client/ios/FxAClient/FxAccountManager.swift:554:1: warning: (redundantInternal) Remove redundant internal access control.
components/fxa-client/ios/FxAClient/FxAccountManager.swift:565:1: warning: (redundantInternal) Remove redundant internal access control.
components/fxa-client/ios/FxAClient/FxAccountManager.swift:587:1: warning: (redundantInternal) Remove redundant internal access control.
components/fxa-client/ios/FxAClient/FxAccountManager.swift:594:1: warning: (redundantInternal) Remove redundant internal access control.
components/fxa-client/ios/FxAClient/FxAccountManager.swift:630:1: warning: (redundantInternal) Remove redundant internal access control.
SwiftFormat completed in 0.84s.
```

┆Issue is synchronized with this [Jira Task](https://mozilla-hub.atlassian.net/browse/SYNC-3883)

--- a/components/fxa-client/ios/FxAClient/FxAccountDeviceConstellation.swift
+++ b/components/fxa-client/ios/FxAClient/FxAccountDeviceConstellation.swift
@@ -130,14 +130,14 @@ public class DeviceConstellation {
 
     /// This allows us to be helpful in certain circumstances e.g. refreshing the device list
     /// if we see a "device disconnected" push notification.
-    internal func processAccountEvent(_ event: AccountEvent) {
+    func processAccountEvent(_ event: AccountEvent) {
         switch event {
         case .deviceDisconnected, .deviceConnected: refreshState()
         default: return
         }
     }
 
-    internal func initDevice(name: String, type: DeviceType, capabilities: [DeviceCapability]) {
+    func initDevice(name: String, type: DeviceType, capabilities: [DeviceCapability]) {
         // This method is called by `FxAccountManager` on its own asynchronous queue, hence
         // no wrapping in a `DispatchQueue.global().async`.
         assert(!Thread.isMainThread)
@@ -148,7 +148,7 @@ public class DeviceConstellation {
         }
     }
 
-    internal func ensureCapabilities(capabilities: [DeviceCapability]) {
+    func ensureCapabilities(capabilities: [DeviceCapability]) {
         // This method is called by `FxAccountManager` on its own asynchronous queue, hence
         // no wrapping in a `DispatchQueue.global().async`.
         assert(!Thread.isMainThread)

--- a/components/fxa-client/ios/FxAClient/FxAccountLogging.swift
+++ b/components/fxa-client/ios/FxAClient/FxAccountLogging.swift
@@ -11,15 +11,15 @@ class FxALog {
         category: "FxAccountManager"
     )
 
-    internal static func info(_ msg: String) {
+    static func info(_ msg: String) {
         log(msg, type: .info)
     }
 
-    internal static func debug(_ msg: String) {
+    static func debug(_ msg: String) {
         log(msg, type: .debug)
     }
 
-    internal static func error(_ msg: String) {
+    static func error(_ msg: String) {
         log(msg, type: .error)
     }
 

--- a/components/fxa-client/ios/FxAClient/FxAccountManager.swift
+++ b/components/fxa-client/ios/FxAClient/FxAccountManager.swift
@@ -317,7 +317,7 @@ open class FxAccountManager {
 
     let fxaFsmQueue = DispatchQueue(label: "com.mozilla.fxa-mgr-queue")
 
-    internal func processEvent(event: Event, completionHandler: @escaping () -> Void) {
+    func processEvent(event: Event, completionHandler: @escaping () -> Void) {
         fxaFsmQueue.async {
             var toProcess: Event? = event
             while let evt = toProcess {
@@ -340,7 +340,7 @@ open class FxAccountManager {
     }
 
     // swiftlint:disable function_body_length
-    internal func stateActions(forState: AccountState, via: Event) -> Event? {
+    func stateActions(forState: AccountState, via: Event) -> Event? {
         switch forState {
         case .start: do {
                 switch via {
@@ -539,19 +539,19 @@ open class FxAccountManager {
         return nil
     }
 
-    internal func createAccount() -> PersistedFirefoxAccount {
+    func createAccount() -> PersistedFirefoxAccount {
         return PersistedFirefoxAccount(config: config.rustConfig)
     }
 
-    internal func tryRestoreAccount() -> PersistedFirefoxAccount? {
+    func tryRestoreAccount() -> PersistedFirefoxAccount? {
         return accountStorage.read()
     }
 
-    internal func makeDeviceConstellation(account: PersistedFirefoxAccount) -> DeviceConstellation {
+    func makeDeviceConstellation(account: PersistedFirefoxAccount) -> DeviceConstellation {
         return DeviceConstellation(account: account)
     }
 
-    internal func postAuthenticated(authType: FxaAuthType) {
+    func postAuthenticated(authType: FxaAuthType) {
         DispatchQueue.main.async {
             NotificationCenter.default.post(
                 name: .accountAuthenticated,
@@ -562,7 +562,7 @@ open class FxAccountManager {
         requireConstellation().refreshState()
     }
 
-    internal func setupInternalListeners() {
+    func setupInternalListeners() {
         // Handle auth exceptions caught in classes that don't hold a reference to the manager.
         _ = NotificationCenter.default.addObserver(forName: .accountAuthException, object: nil, queue: nil) { _ in
             self.processEvent(event: .authenticationError) {}
@@ -584,14 +584,14 @@ open class FxAccountManager {
         }
     }
 
-    internal func requireAccount() -> PersistedFirefoxAccount {
+    func requireAccount() -> PersistedFirefoxAccount {
         if let acct = account {
             return acct
         }
         preconditionFailure("initialize() must be called first.")
     }
 
-    internal func requireConstellation() -> DeviceConstellation {
+    func requireConstellation() -> DeviceConstellation {
         if let cstl = constellation {
             return cstl
         }
@@ -627,7 +627,7 @@ public enum FxaAuthType {
     case recovered
     case other(reason: String)
 
-    internal static func fromActionQueryParam(_ action: String) -> FxaAuthType {
+    static func fromActionQueryParam(_ action: String) -> FxaAuthType {
         switch action {
         case "signin": return .signin
         case "signup": return .signup

--- a/components/fxa-client/ios/FxAClient/FxAccountState.swift
+++ b/components/fxa-client/ios/FxAClient/FxAccountState.swift
@@ -7,7 +7,7 @@ import Foundation
 /**
  * States of the [FxAccountManager].
  */
-internal enum AccountState {
+enum AccountState {
     case start
     case notAuthenticated
     case authenticationProblem
@@ -19,7 +19,7 @@ internal enum AccountState {
  * Base class for [FxAccountManager] state machine events.
  * Events aren't a simple enum class because we might want to pass data along with some of the events.
  */
-internal enum Event {
+enum Event {
     case initialize
     case accountNotFound
     case accountRestored

--- a/components/fxa-client/ios/FxAClient/FxAccountStorage.swift
+++ b/components/fxa-client/ios/FxAClient/FxAccountStorage.swift
@@ -5,9 +5,9 @@
 import Foundation
 
 class KeyChainAccountStorage {
-    internal var keychainWrapper: MZKeychainWrapper
-    internal static var keychainKey: String = "accountJSON"
-    internal static var accessibility: MZKeychainItemAccessibility = .afterFirstUnlock
+    var keychainWrapper: MZKeychainWrapper
+    static var keychainKey: String = "accountJSON"
+    static var accessibility: MZKeychainItemAccessibility = .afterFirstUnlock
 
     init(keychainAccessGroup: String?) {
         keychainWrapper = MZKeychainWrapper.sharedAppContainerKeychain(keychainAccessGroup: keychainAccessGroup)

--- a/components/fxa-client/ios/FxAClient/MZKeychain/KeychainItemAccessibility.swift
+++ b/components/fxa-client/ios/FxAClient/MZKeychain/KeychainItemAccessibility.swift
@@ -1,5 +1,5 @@
 //
-//  KeychainOptions.swift
+//  KeychainItemAccessibility.swift
 //  SwiftKeychainWrapper
 //
 //  Created by James Blair on 4/24/16.

--- a/components/fxa-client/ios/FxAClient/PersistedFirefoxAccount.swift
+++ b/components/fxa-client/ios/FxAClient/PersistedFirefoxAccount.swift
@@ -257,7 +257,7 @@ class PersistedFirefoxAccount {
         }
     }
 
-    internal func notifyAuthErrors<T>(_ cb: () throws -> T) rethrows -> T {
+    func notifyAuthErrors<T>(_ cb: () throws -> T) rethrows -> T {
         do {
             return try cb()
         } catch let error as FxaError {
@@ -269,7 +269,7 @@ class PersistedFirefoxAccount {
         }
     }
 
-    internal func notifyAuthError() {
+    func notifyAuthError() {
         NotificationCenter.default.post(name: .accountAuthException, object: nil)
     }
 }

--- a/components/nimbus/ios/Nimbus/Dictionary+.swift
+++ b/components/nimbus/ios/Nimbus/Dictionary+.swift
@@ -4,7 +4,7 @@
 
 import Foundation
 
-internal extension Dictionary where Key == String, Value == Any {
+extension Dictionary where Key == String, Value == Any {
     func stringify() throws -> String {
         let data = try JSONSerialization.data(withJSONObject: self)
         guard let s = String(data: data, encoding: .utf8) else {

--- a/components/nimbus/ios/Nimbus/FeatureVariables.swift
+++ b/components/nimbus/ios/Nimbus/FeatureVariables.swift
@@ -303,9 +303,9 @@ extension VariablesWithBundle {
 
 /// A thin wrapper around the JSON produced by the `get_feature_variables_json(feature_id)` call, useful
 /// for configuring a feature, but without needing the developer to know about experiment specifics.
-internal class JSONVariables: VariablesWithBundle {
+class JSONVariables: VariablesWithBundle {
     private let json: [String: Any]
-    internal let resourceBundles: [Bundle]
+    let resourceBundles: [Bundle]
 
     init(with json: [String: Any], in bundles: [Bundle] = [Bundle.main]) {
         self.json = json

--- a/components/nimbus/ios/Nimbus/Nimbus.swift
+++ b/components/nimbus/ios/Nimbus/Nimbus.swift
@@ -26,9 +26,9 @@ public class Nimbus: NimbusInterface {
         return queue
     }()
 
-    internal init(nimbusClient: NimbusClientProtocol,
-                  resourceBundles: [Bundle],
-                  errorReporter: @escaping NimbusErrorReporter)
+    init(nimbusClient: NimbusClientProtocol,
+         resourceBundles: [Bundle],
+         errorReporter: @escaping NimbusErrorReporter)
     {
         self.errorReporter = errorReporter
         self.nimbusClient = nimbusClient
@@ -151,13 +151,13 @@ extension Nimbus: FeaturesInterface {
         ))
     }
 
-    internal func getEnrollmentByFeature(featureId: String) -> EnrolledFeature? {
+    func getEnrollmentByFeature(featureId: String) -> EnrolledFeature? {
         return catchAll {
             try nimbusClient.getEnrollmentByFeature(featureId: featureId)
         }
     }
 
-    internal func postEnrollmentCalculation(_ events: [EnrollmentChangeEvent]) {
+    func postEnrollmentCalculation(_ events: [EnrollmentChangeEvent]) {
         // We need to update the experiment enrollment annotations in Glean
         // regardless of whether we recieved any events. Calling the
         // `setExperimentActive` function multiple times with the same
@@ -172,7 +172,7 @@ extension Nimbus: FeaturesInterface {
         notifyOnExperimentsApplied(experiments)
     }
 
-    internal func recordExperimentTelemetry(_ experiments: [EnrolledExperiment]) {
+    func recordExperimentTelemetry(_ experiments: [EnrolledExperiment]) {
         for experiment in experiments {
             Glean.shared.setExperimentActive(
                 experiment.slug,
@@ -182,7 +182,7 @@ extension Nimbus: FeaturesInterface {
         }
     }
 
-    internal func recordExperimentEvents(_ events: [EnrollmentChangeEvent]) {
+    func recordExperimentEvents(_ events: [EnrollmentChangeEvent]) {
         for event in events {
             switch event.change {
             case .enrollment:
@@ -219,7 +219,7 @@ extension Nimbus: FeaturesInterface {
         }
     }
 
-    internal func getFeatureConfigVariablesJson(featureId: String) -> [String: Any]? {
+    func getFeatureConfigVariablesJson(featureId: String) -> [String: Any]? {
         do {
             guard let string = try nimbusClient.getFeatureConfigVariables(featureId: featureId) else {
                 return nil
@@ -264,7 +264,7 @@ private extension Nimbus {
 /*
  * Methods split out onto a separate internal extension for testing purposes.
  */
-internal extension Nimbus {
+extension Nimbus {
     func setGlobalUserParticipationOnThisThread(_ value: Bool) throws {
         let changes = try nimbusClient.setGlobalUserParticipation(optIn: value)
         postEnrollmentCalculation(changes)

--- a/components/nimbus/ios/Nimbus/NimbusMessagingHelpers.swift
+++ b/components/nimbus/ios/Nimbus/NimbusMessagingHelpers.swift
@@ -26,7 +26,7 @@ public protocol NimbusMessagingProtocol {
 // Deprecated the name GleanPlumb.
 public typealias GleanPlumbProtocol = NimbusMessagingProtocol
 public typealias GleanPlumbMessageHelper = NimbusMessagingHelper
-public typealias NimbusMessagingHelperProtocol = NimbusTargetingHelperProtocol & NimbusStringHelperProtocol
+public typealias NimbusMessagingHelperProtocol = NimbusStringHelperProtocol & NimbusTargetingHelperProtocol
 
 /**
  * A helper object to make working with Strings uniform across multiple implementations of the messaging
@@ -61,7 +61,7 @@ public class NimbusMessagingHelper: NimbusMessagingHelperProtocol {
 
 // MARK: Dummy implementations
 
-internal class AlwaysConstantTargetingHelper: NimbusTargetingHelperProtocol {
+class AlwaysConstantTargetingHelper: NimbusTargetingHelperProtocol {
     private let constant: Bool
 
     public init(constant: Bool = false) {
@@ -73,7 +73,7 @@ internal class AlwaysConstantTargetingHelper: NimbusTargetingHelperProtocol {
     }
 }
 
-internal class EchoStringHelper: NimbusStringHelperProtocol {
+class EchoStringHelper: NimbusStringHelperProtocol {
     public func getUuid(template _: String) -> String? {
         nil
     }

--- a/components/places/ios/Places/Places.swift
+++ b/components/places/ios/Places/Places.swift
@@ -5,8 +5,8 @@
 import Foundation
 import os.log
 
-internal typealias UniffiPlacesApi = PlacesApi
-internal typealias UniffiPlacesConnection = PlacesConnection
+typealias UniffiPlacesApi = PlacesApi
+typealias UniffiPlacesConnection = PlacesConnection
 
 /**
  * This is specifically for throwing when there is

